src/components/index.ts

```diff
@@ -14,6 +14,9 @@ export * from "./Image";
 export * from "./LeftNavigation";
 export * from "./List";
 export * from "./ListItem";
+export * from "./Navbar";
+export * from "./Navbar/LeftContent";
+export * from "./Navbar/RightContent";
```

Comment:
I argue that this should simplify to
```suggestion
export * from "./Navbar";
```

Navbar should take care of its own dependancies.

Reply:
I'll need to add the other deps to the index at the Navbar level for that to work so I'll do that.